heatx.tex

```diff
@@ -440,6 +406,100 @@ \subsection{Computation of KERMA Factors By Energy Balance}
 $\overline{E}_\gamma$, $Y_\gamma$, and $\sigma_\gamma$ is
 subtracted from the neutron contribution.
 
+\subsubsection{The special case of fission}
+\label{sssHEATR_fissionKERMA}
+
+The partial KERMA for fission is a special case due to the particular
```

Comment:
Wherever we have `Q value`, it should be changed to `$Q$-value`. `Q` should also be `$Q$` everywhere so that it matches the variable in the equations.